SmartDeviceLink_Example/Enum+SDLConnectionType.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Enum+SDLConnectionType.swift
+//  SmartDeviceLink-Example-Swift
+//
+//  Created by Nicole on 4/10/18.
+//  Copyright © 2018 smartdevicelink. All rights reserved.
+//
+
+import Foundation
+
+enum SDLConnectionType {
```

Comment:
This can just be in the proxy class, they should all start lower case (tcp / iap is fine)